mesonbuild/dependencies/base.py

```diff
@@ -693,10 +693,15 @@ def _call_pkgbin(self, args, env=None):
         sysroot = self.env.properties[self.for_machine].get_sys_root()
         if sysroot:
             env['PKG_CONFIG_SYSROOT_DIR'] = sysroot
-        new_pkg_config_path = ':'.join([p for p in extra_paths])
+        new_pkg_config_path = os.pathsep.join([p for p in extra_paths])
```

Comment:
I know it's in the original code, but this really should be:
```python
os.pathsep.join(extra_paths)
```

Reply:
lol good catch :+1: